src/main/java/com/github/dockerjava/api/model/InternetProtocol.java

```diff
@@ -2,15 +2,19 @@
 
 /**
  * The IP protocols supported by Docker.
- * 
+ *
  * @see #TCP
  * @see #UDP
  */
 public enum InternetProtocol {
-    /** The <i>Transmission Control Protocol</i> */
+    /**
+     * The <i>Transmission Control Protocol</i>
```

Comment:
probably this was caused by my autoformatter when i wanted remove redundant space. Let's keep as is.